modules/tinymce/src/core/main/ts/api/dom/Sizzle.ts

```diff
@@ -1442,9 +1442,11 @@ Expr = Sizzle.selectors = {
 Expr.pseudos.nth = Expr.pseudos.eq;
 
 // Add button/input type pseudos
+// eslint-disable-next-line guard-for-in
 for (i in { radio: true, checkbox: true, file: true, password: true, image: true }) {
```

Comment:
`Arr.each(["radio", "checkbox", "etc"], (i) => {..})` would be a better pattern and avoid needing to disable the warning 🤔

Reply:
I will do this but we currently seem to treat Sizzle like external code so the existing practice seems to have been disabling lints when needed but otherwise not touching the code.

Reply:
Yeah, that's my understanding as well. To be honest, we should probably just disable all eslint checks for the whole file.

Reply:
oh, sorry, I didn't notice which file it was in. That's fine.

Reply:
Done

Reply:
> oh, sorry, I didn't notice which file it was in. You're right, we have vendored sizzle. That's fine.
Due to Githubs dodgy comment updating I didn't notice your reply before making the change. Should I rollback?